internal/hcs/process.go

```diff
@@ -276,15 +276,20 @@ func (process *Process) ExitCode() (_ int, err error) {
 
 	properties, err := process.Properties()
 	if err != nil {
-		return 0, makeProcessError(process, operation, err, nil)
+		return -1, makeProcessError(process, operation, err, nil)
```

Comment:
@jhowardmsft - Is it safe to change all of these or just the `LastWaitResult` condition?

Reply:
Pretty sure this is safe.